13-ncvs-vignette.Rmd

```diff
@@ -610,11 +610,11 @@ Now that we have prepared our data and created the design objects, we can calcul
 
 3. *Victimization rates* are estimates of the number of victimizations per 1,000 persons or households in the population.
 
-4. Prevalence rates are estimates of the percentage of the population (persons or households) who are victims of a crime.
+4. *Prevalence rates* are estimates of the percentage of the population (persons or households) who are victims of a crime.
 
 ### Estimation 1: Victimization Totals {#vic-tot}
 
-There are two ways to calculate victimization totals. Using the incident design object (`inc_des`) is the most straightforward method, but the person (`pers_des`) and household (`hh_des`) design objects can be used as well if the adjustment factor (`ADJINC_WT`) is incorporated. In the example below, the total number of property and violent victimizations is first calculated using the incident file and then using the household and person design objects. The incident file is smaller, and thus, estimation is faster using that file, but the estimates will be the same as illustrated below:
+There are two ways to calculate victimization totals. Using the incident design object (`inc_des`) is the most straightforward method, but the person (`pers_des`) and household (`hh_des`) design objects can be used as well if the adjustment factor (`ADJINC_WT`) is incorporated. In the example below, the total number of property and violent victimizations is first calculated using the incident file and then using the household and person design objects. The incident file is smaller, and thus, estimation is faster using that file, but the estimates are the same as illustrated below:
 
 ```{r}
```

Comment:
Should the output for these tibbles be gt? they look a little odd in the PDF version of the book as just code outputs.

Reply:
Updated to be gt output